scripts/release_scripts/repo_specific_changes_fetcher.py

```
@@ -157,3 +162,17 @@ def get_changes(release_tag_to_diff_against):
             changes.append('* %s\n' % item)
 
     return changes
+
+
+def main(args=None):
+    """Main method for fetching repos specific changes."""
+    options = _PARSER.parse_args(args=args)
+    changes = get_changes(options.release_tag)
```

Are we expecting to run this script always with release_tag?  if yes, should we raise error if not provided?

---

I have added `required` to the `--release_tag`.